apps/analytics/views.py

```diff
@@ -108,11 +108,9 @@ def get_submission_count(request, challenge_pk, duration):
 @api_view(['GET', ])
 @permission_classes((permissions.IsAuthenticated, HasVerifiedEmail, IsChallengeCreator))
 @authentication_classes((ExpiringTokenAuthentication,))
-def get_challenge_phase_submission_analysis(request, challenge_pk, challenge_phase_pk):
+def get_challenge_phase_submission_count_by_team(request, challenge_pk, challenge_phase_pk):
     """
-    API to fetch
-    1. The submissions count for challenge phase.
-    2. The participated team count for challenge phase.
+    API to fetch the submissions count in a challenge phase for a participant team.
```

Comment:
Please replace this with `Returns number of submissions done by a participant team in a challenge phase`